src/main/java/Problem3/Book.java

```diff
@@ -19,29 +19,61 @@ public Book(Book anotherBook) {
         this.author = anotherBook.author;
     }
 
+
+    public void setId(UUID newId) {
+        this.id = newId;
+    }
+
+    public void setTitle(String newTitle) {
+        this.title = newTitle;
+    }
+
+    public void setAuthor(String newAuthor) {
+        this.author = newAuthor;
+    }
+
     @Override
     public boolean equals(Object obj) {
+/*
+        System.out.println("I am testing to see org book is equal too ");
+        System.out.println("Test " + this);
+        System.out.println("Test obj " + obj);
+        System.out.println("------------------");
+*/
         if (obj == this) {
             return true;
         }
-
+/*
+        System.out.println("I am testing to see org book id and title ");
+        System.out.println("What is book?  " + id);
+        System.out.println("What is book?  " + title);
+        System.out.println("------------------");
+*/
         if (!(obj instanceof Book)) {
```

Comment:
For more efficient code, try 
```suggestion
        return (obj == this);
```